### DISCUSSION TITLE
Remove helmrelease from Flux without removing resources from cluster

### DISCUSSION THREAD

**Discussion by johnlane** [fluxcd/flux2#5017] [Q&A]
Hello, I have an old version of Flux (0.26.3) with a number of Helm releases on it. I need to remove one of them from Flux without distroying any resources.

I read [this discussion](https://github.com/fluxcd/flux2/discussions/2790) and used `flux suspend helmrelease` before removing the release from Git (removing from resources section of `kustomization.yaml`). However this went ahead and destroyed the resources.

Is there a way to do this, to remove a chart from Flux but leave it untouched in the cluster?

**Comment by johnlane** [comment]
Just for the benefit of anyone else having this query.

1. Suspend the helmrelease.
2. Scale the `kustomize-controller` down to zero replicas.
3. Remove helmrelease from Git; at least from `kustomization.yaml`. Merge and push.
4. Scale the `kustomize-controller` up.
5. The helmrelease should disappear from  `flux get helmrelease` but the resources will remain.

At least that worked for me.

**Reply by stefanprodan** [reply]
Suspending the HelmRelease works Ok with all supported versions of Flux 2.2, 2.3 and 2.4.

**Reply by johnlane** [reply]
Yeah, I was on an ancient version `0.26.3`, unfortunately.
